lib/Dispatcher.js

```diff
@@ -39,29 +47,35 @@ module.exports = function () {
      * @throws {Error} if store is invalid
      * @throws {Error} if store is already registered
      */
-    Dispatcher.registerStore = function registerStore(store) {
-        if ('function' !== typeof store) {
+    registerStore: function registerStore(store) {
+        if (!BaseStore.isPrototypeOf(store)) {
```

Comment:
in the previous case any function would be allowed, now we guaranty only object that inherit from the BaseStore.